examples/load_keys.c

```diff
@@ -1,27 +1,89 @@
 #include "transbank.h"
 #include "transbank_serial_utils.h"
 
-int main() {
+int main()
+{
+  // VELOCIDAD COM
+  const int bRate = 115200;
+  const char sep = '|';
 
-  char* portName = "COM4";
-  int retval = open_port(portName, 115200);
-  if ( retval == TBK_OK ){
-    puts("Serial port successfully opened.\n");
+  // Getting available ports
+  char *list = strcat(list_ports(), "|"); // Add pipe for examples
 
-    BaseResponse rsp = load_keys();
+  int x, count = 0;
+  for (x = 0; x <= strlen(list); x++)
+  {
+    if (list[x] == sep)
+    {
+      count++;
+    }
+  }
+
+  char **ports = malloc(count * sizeof(char *));
+  char *w;
+  int i, l, f = 0;
+
+  printf("Available ports : \n=================\n");
+  for (x = 0; x <= strlen(list); x++)
+  {
+    if (list[x] == sep)
+    {
+      w = malloc(l * sizeof(char *));
+      strncpy(w, list + i, l);
+      w[l] = 0;
+
+      printf("%i) %s\n", (f + 1), w); // Display option
+
+      // Store port (w) in array list
+      ports[f] = malloc(strlen(w) * sizeof(char *));
+      strcpy(ports[f], w);
+
+      f++;
+      i = x + 1; // Set new position
+      l = 0;
+      continue;
+    }
+
+    l++;
+  }
+
+  // Handle input option
+  printf("Please insert port option:");
+  int op;
+  scanf("%i", &op);
+
+  char *portName = ports[op - 1];
+  printf("Using option: %i -> \"%s\" port ...\n=================\n", op, portName);
+
+  int resp = open_port(portName, bRate);
+  if (resp == TBK_OK)
```

Comment:
Sounds that all the logic to show the available ports and select which of theme to open, can be put in one single file or method, so we can avoid duplicate code.